package.json

```diff
@@ -37,7 +37,7 @@
   "main": "d2l-localize-behavior.js",
   "dependencies": {
     "@brightspace-ui/intl": "^3",
-    "@polymer/app-localize-behavior": "^3.0.1",
+    "@polymer/app-localize-behavior": "BrightspaceUI/app-localize-behavior#v3.0.1-d2l",
```

Comment:
Does this cause a problem if an FRA is using this directly, and pulling in some of our old Polymer shared components?  Or are multiple versions of a behaviour going to be properly included by their bundling?

Reply:
That's a good question. I found 5 FRAs that were using `app-localize-behavior` (v3) directly and that could be using one of our components that would be relying on `localize-behavior`. I'm not totally sure what would happen... they might get nested copies which wouldn't be great, and then they'd need to add a resolution.

Reply:
For reference the 5 places where this could be a problem:
- [manager-view-fra](https://github.com/Brightspace/manager-view-fra/blob/70577bf6b7dd4999c0b87b87e85627e1ac85157d/package.json#L77)
- [d2l-activity-details-ui](https://github.com/Brightspace/d2l-activity-details-ui/blob/3d616ea18b625f7eec60cc3b0a592ad538e9d180/package.json#L42)
- [d2l-recent-grades-ui](https://github.com/Brightspace/d2l-recent-grades-ui/blob/f5fed7f8c4cc835e0ee396d537ad96b1752a5098/package.json#L60)
- [user-switcher](https://github.com/Brightspace/user-switcher/blob/102fda6f254d10f7366d863bf543d0674815f754/package.json#L50)
- [d2l-richtext-parent](https://github.com/Brightspace/d2l-richtext-parent/blob/69d01b81d463f75b61fbfd0a5c397188d007ec77/package.json#L47)

Reply:
Updated the description with the places using `app-localize-behavior`, will track fixes as they go in and wait to merge this until they're ready.